### PR TITLE
Build RISC-V target only for -next

### PR DESCRIPTION
**Pull request by tpimh**
As suggested in https://github.com/ClangBuiltLinux/continuous-integration/pull/70#issuecomment-599714608, RISC-V targets are only enabled on linux-next, so they would build fine without the patches. Build logs for these targets can be found [here](https://github.com/ClangBuiltLinux/continuous-integration/runs/512210549).

### CODE REVIEW

File: driver.sh [hunk 1]
```diff
@@ -149,6 +149,20 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "riscv32")
+      config=defconfig
+      image_name=vmlinux
+      using_qemu=false
+      export ARCH=riscv
+      export CROSS_COMPILE=riscv64-linux-gnu- ;;
```
Comment (nathanchance) [inline driver.sh]: Should this be `riscv32-linux-gnu-`?

File: driver.sh [hunk 2]
```diff
@@ -149,6 +149,20 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "riscv32")
+      config=defconfig
```
Comment (nathanchance) [inline driver.sh]: Should this be `rv32_defconfig`?

File: driver.sh (hunk 1 above)
Comment (tpimh) [reply driver.sh]: I don't think debian even has riscv32 binutils in the repo.

File: driver.sh (hunk 2 above)
Comment (tpimh) [reply driver.sh]: Probably these two issues are the reason why the kernels produced are nearly identical: https://github.com/ClangBuiltLinux/continuous-integration/pull/70#issuecomment-599708751

File: driver.sh (hunk 1 above)
Comment (nathanchance) [reply driver.sh]: Doesn't look like it. Seems like we should only be building for riscv64 then.

Comment (nickdesaulniers) [reply driver.sh]: @palmer-dabbelt , any recommendations on 32b riscv binutils?

**Pull request merged**